Configurations/unix-Makefile.tmpl

```diff
@@ -310,6 +310,14 @@ MODULESDIR=$(libdir)/ossl-modules
 # libraries and applications
 LIBRPATH=$(libdir)
 
+BINDIR={- our $bindir = $config{bindir};
+          unless ($bindir) {
+              $bindir = "bin$target{multibin}";
+          }
+          file_name_is_absolute($bindir) ? "" : $bindir -}
+bindir={- file_name_is_absolute($bindir)
+          ? $bindir : '$(INSTALLTOP)/$(BINDIR)' -}
```

Comment:
Allowing `bindir` to be absolute is a very bad idea.  Generally speaking, it's a sure way to break the expectations of using DESTDIR.

Our documentation of this variable is unfortunately not very well documented, probably because we've taken knowledge of it for granted (as far as I know, it's originally an FSF idea, but has spread since).

Reply:
https://github.com/openssl/openssl/blob/e81c81c9af8a5d22658110d2dc753582eb87a58e/INSTALL.md?plain=1#L1338-L1346

Reply:
Looking at this again with fresh eyes, I retract my concerns...  this idea doesn't look too bad after all